src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -1,189 +1,95 @@
-            CANDLE.setLEDs(thirdSectionColor.getRed(), thirdSectionColor.getGreen(), thirdSectionColor.getBlue(), 0, offset + firstLEDCount + secondLEDCount, thirdLEDCount);
-        } else {
-            CANDLE.setLEDs(thirdSectionColor.getRed(), thirdSectionColor.getGreen(), thirdSectionColor.getBlue(), 0, offset, firstLEDCount);
-            CANDLE.setLEDs(secondSectionColor.getRed(), secondSectionColor.getGreen(), secondSectionColor.getBlue(), 0, offset + firstLEDCount, secondLEDCount);
-            CANDLE.setLEDs(firstSectionColor.getRed(), firstSectionColor.getGreen(), firstSectionColor.getBlue(), 0, offset + firstLEDCount + secondLEDCount, thirdLEDCount);
-        }
+    public int getNumberOfLEDS() {
+        return numberOfLEDs;
     }
 
-    void animateFire(double brightness, double speed, double sparking, double cooling) {
-        CANDLE.animate(
-                new FireAnimation(
-                        brightness,
-                        speed,
-                        this.numberOfLEDs,
-                        sparking,
-                        cooling,
-                        inverted,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void clearLedColors() {
+        setAllStripColors(Color.kBlack);
     }
 
-    void animateRainbow(double brightness, double speed) {
-        CANDLE.animate(
-                new RainbowAnimation(
-                        brightness,
-                        speed,
-                        this.numberOfLEDs,
-                        inverted,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void staticColor(Color color) {
+        setAllStripColors(color);
     }
 
-    void animateColorFlow(Color color, double speed) {
-        CANDLE.animate(new ColorFlowAnimation(
-                        color.getRed(),
-                        color.getGreen(),
-                        color.getBlue(),
-                        0,
-                        speed,
-                        this.numberOfLEDs,
-                        inverted ? ColorFlowAnimation.Direction.Backward : ColorFlowAnimation.Direction.Forward,
-                        this.offset
-                ),
-                animationSlot
-        );
-    }
 
-    void animateLarson(Color color, double speed, LarsonAnimation.BounceMode mode, int size) {
-        CANDLE.animate(
-                new LarsonAnimation(
-                        color.getRed(),
-                        color.getGreen(),
-                        color.getBlue(),
-                        0,
-                        speed,
-                        this.numberOfLEDs,
-                        mode,
-                        size,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void blink(Color color) {
```

Comment:
It's probably better to have the blinking interval as a parameter so we can have some things slow blink and some fast.